clients/feeder/feeder.go

```diff
@@ -171,10 +172,21 @@ func (c *Client) get(ctx context.Context, queryURL string) ([]byte, error) {
 			}
 
 			res, err = c.client.Do(req)
-			if err == nil && res != nil && res.StatusCode == http.StatusOK {
-				return io.ReadAll(res.Body)
-			} else if res != nil && res.StatusCode != http.StatusOK {
-				err = errors.New(res.Status)
+			if err == nil {
+				if res.StatusCode == http.StatusOK {
+					resBytes, err = io.ReadAll(res.Body)
+					if err != nil {
+						return nil, err
```

Comment:
Close missing here?

I think we can just defer `res.Body.Close()` if `err == nil`

Reply:
> I think we can just defer res.Body.Close() if err == nil

lol, triggers another lint error.